README.md

```diff
@@ -1,2 +1,26 @@
-# SOURCE-AASF
-Contain details of all projects and general contributing guideline to any project in Source-AASF.
+# Source-AASF
+
+Contain details of all projects under Source-AASF.
```

Comment:
change repo name to aasf iiitm or may be change this repo name itself to source aasf